frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1055,6 +1064,45 @@ pub mod pallet {
 			<QueuedSolution<T>>::put(solution);
 			Ok(())
 		}
+
+		#[pallet::weight(100)]
+		pub fn challenge_solution(origin: OriginFor<T>, index: u32) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+			// TODO: Create custom error
+			ensure!(
+				T::Currency::can_slash(&who, T::MinimumSlashableAmount::get()),
+				<Error<T>>::CallNotAllowed
+			);
+			// TODO: Add an `else` for a None result
+			if let Some(submission) = SignedSubmissionsMap::<T>::get(index) {
+				match Self::feasibility_check(
+					submission.raw_solution.clone(),
+					ElectionCompute::Signed,
+				) {
+					Ok(solution) => {
+						let _ = T::Currency::slash(&who, T::MinimumSlashableAmount::get());
```

Comment:
Like Kian mention, `T::MinimumSlashableAmount::get()` should be replaced with the deposit for the solution